source/blog/2016-01-11-ember-data-2-3-released.md

```diff
@@ -0,0 +1,196 @@
+
+// DS.attr
+import attr from 'ember-data/attr';
+
+// DS.hasMany or DS.belongsTo
+import { hasMany, belongsTo } from 'ember-data/relationships';
+```
+
+Modules that are not considered to be public API have been placed under the `ember-data/-private` path. Please do not import these modules into your app or test code as they may be changed or removed *without warning*.
+
+Thanks to [@rwjblue](https://github.com/rwjblue) for implementing this
+module API.
+
+#### Notable Bug Fixes
+
+##### [PR #4025 Use keyForRelationship for belongsTo and hasMany](https://github.com/emberjs/data/pull/4025/)
+The `EmbeddedRecordsMixin` now uses [keyForRelationship](http://emberjs.com/api/data/classes/DS.RESTSerializer.html#method_keyForRelationship) to generate the serialized key for embedded relationships. Thanks to [@GCorbel](https://github.com/GCorbel) for this fix.
+
+##### [PR #3866 Allow store.push to accept { data: null }](https://github.com/emberjs/data/pull/3866)
+The JSONAPISerializer now correctly accepts `{"data": null}` as a valid response instead of throwing an unhelpful error message. Thanks to [@mitchlloyd](https://github.com/mitchlloyd) for the [bug report](https://github.com/emberjs/data/issues/3790).
+
+
+See the [Ember Data 2.3.0 CHANGELOG](https://github.com/emberjs/data/blob/v2.3.0/CHANGELOG.md) for a full list of all changes.
+
+
+### Ember Data 2.4 beta
+
+Ember Data 2.4 beta is released today, and in six weeks it will become the
+new stable version of Ember Data. This beta cycle introduces a single new feature.
+
+#### Friendly Errors
+
+[@nikz](https://github.com/nikz) has implemented
+[RFC 101](https://github.com/emberjs/rfcs/pull/101) which provides
+more context for `RESTAdapter` and `JSONAPIAdapter` errors in Ember
+Data. Be sure to check out the
+[pull request](https://github.com/emberjs/data/pull/3930) for more
+details.
+
+For more details on changes in 2.4, review the
+[Ember Data 2.4.0-beta.1 CHANGELOG](https://github.com/emberjs/data/blob/v2.4.0-beta.1/CHANGELOG.md).
+
+
+### Upcomming Features
+
+Two new features recently landed on the Ember Data canary branch. They
+each address add some long requested features to Ember Data. However,
+before they can be enabled in a beta branch the Ember Data team would
+like the community to try them out and provide feedback on their
+implementations. These two feature flaged features are only available
+on the master branch (sometimes called "canary") of Ember Data. To
+test them out please update the version of Ember Data in package.json
+to `emberjs/data#master` and add the feature to the `EmberENV`'s
+`FEATURES` object in `config/environment.js`.
+
+```config/environment.js
+var ENV = {
+  EmberENV: {
+    FEATURES: {
+      'ds-find-include': true,
+      'ds-references': true
+    }
+  }
+};
+```
+
+For more information you can check out Ember's
+[feature flags guide](https://guides.emberjs.com/v2.2.0/configuring-ember/feature-flags/).
+
+#### `ds-find-include`
```

Comment:
The feature is `ds-finder-include`, as pointed out by @HeroicEric https://github.com/emberjs/data/pull/4059#issuecomment-170882397. My PR linked below updates this...